### PR TITLE
autopep8

### PR DESCRIPTION
**Pull request by mlautman**
Results of running 
`autopep8 --max-line-length 120 --in-place $1` on each `.py` file

### CODE REVIEW

File: rqt_gui/src/rqt_gui/ros_plugin_provider.py [hunk 1]
```diff
@@ -82,12 +82,14 @@ def load(self, plugin_id, plugin_context):
             qCritical('RosPluginProvider.load(%s): raised an exception:\n%s' % (plugin_id, e))
             return None
         except Exception as e:
-            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (plugin_id, attributes['module_name'], attributes['class_from_class_type'], traceback.format_exc()))
+            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (
```
Comment (brawner) [inline rqt_gui/src/rqt_gui/ros_plugin_provider.py:86]: break into more lines

File: rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py [hunk 2]
```diff
@@ -69,7 +69,8 @@ def _check_for_master(self):
         # spawn thread to detect when master becomes available
         self._wait_for_master_thread = threading.Thread(target=self._wait_for_master)
         self._wait_for_master_thread.start()
-        self._wait_for_master_dialog = QMessageBox(QMessageBox.Question, self.tr('Waiting for ROS master'), self.tr("Could not find ROS master. Either start a 'roscore' or abort loading the plugin."), QMessageBox.Abort)
+        self._wait_for_master_dialog = QMessageBox(QMessageBox.Question, self.tr('Waiting for ROS master'), self.tr(
```
Comment (brawner) [inline rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py]: bad line break

File: rqt_py_common/src/rqt_py_common/layout_util.py [hunk 3]
```diff
@@ -66,9 +66,9 @@ def alternate_color(list_widgets, colors_alter=[Qt.white, Qt.gray]):
             i_widget += 1
 
             rospy.logdebug('LayoutUtil divisor={} i_widget={} colors_num={}'.format(
-                                                                   divisor,
-                                                                   i_widget,
-                                                                   colors_num))
+                divisor,
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/layout_util.py]: align with str.format

File: rqt_py_common/src/rqt_py_common/rosaction.py [hunk 4]
```diff
@@ -363,34 +374,39 @@ def rosactionmain(mode=MODE_ACTION):
...
+                                                "expected a mapping node, but found %s" % node.id,
+                                                node.start_mark)
     mapping = collections.OrderedDict()
     for key_node, value_node in node.value:
         key = self.construct_object(key_node, deep=deep)
         if not isinstance(key, collections.Hashable):
             raise yaml.constructor.ConstructorError("while constructing a mapping",
                                                     node.start_mark,
-                    "found unhashable key", key_node.start_mark)
+                                                    "found unhashable key", key_node.start_mark)
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rosaction.py:393]: break for fourth argument

File: rqt_py_common/src/rqt_py_common/rosaction.py [hunk 5]
```diff
@@ -598,7 +616,8 @@ def rosaction_cmd_prototype(args):
         # try to catch the user specifying code-style types and error
         if '::' in message_type:
             if not options.silent:
-                parser.error("rosactionproto does not understand C++-style namespaces (i.e. '::').\nPlease refer to msg/srv types as 'package_name/Type'.")
+                parser.error(
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rosaction.py:618]: break into more lines

File: rqt_py_common/src/rqt_py_common/rqt_ros_graph.py [hunk 6]
```diff
@@ -72,12 +72,12 @@ def get_full_grn(model_index):
             for child_grn in children_grn_list:
                 grn_full = upper_grn + child_grn
                 rospy.logdebug('grn_full={} upper_grn={} child_grn={}'.format(
-                                               grn_full, upper_grn, child_grn))
+                    grn_full, upper_grn, child_grn))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:75]: align with str.format

File: rqt_py_common/src/rqt_py_common/rqt_ros_graph.py [hunk 7]
```diff
@@ -123,12 +123,12 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
         list_grn_children_all = []
         while True:  # Loop per child.
             grn_curr = grn_prev + RqtRosGraph.DELIM_GRN + str(
-                                                            model_index.data())
+                model_index.data())
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_ros_graph.py]: break at +

File: rqt_py_common/src/rqt_py_common/rqt_ros_graph.py [hunk 8]
```diff
@@ -123,12 +123,12 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
...
             child_qmindex = model_index.child(i_child, 0)
 
             if (not child_qmindex.isValid()):
                 rospy.logdebug('!! DEADEND i_child=#{} grn_curr={}'.format(
-                                                           i_child, grn_curr))
+                    i_child, grn_curr))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:131]: align with str.format

File: rqt_py_common/src/rqt_py_common/rqt_ros_graph.py [hunk 9]
```diff
@@ -146,9 +146,9 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
 
             list_grn_children_all = list_grn_children_all + list_grn_children
             rospy.logdebug('111 lennodes={} list_grn_children={}'.format(
-                                len(list_grn_children_all), list_grn_children))
+                len(list_grn_children_all), list_grn_children))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:149]: align with str.format

File: rqt_py_common/src/rqt_py_common/rqt_ros_graph.py [hunk 10]
```diff
@@ -146,9 +146,9 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
...
             rospy.logdebug('122 list_grn_children_all={}'.format(
-                                                        list_grn_children_all))
+                list_grn_children_all))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:151]: align with str.format

File: rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py [hunk 11]
```diff
@@ -69,14 +69,14 @@ def load_parameters(config, caller_id):
             # clear specified parameter namespaces
             # #2468 unify clear params to prevent error
             for param in roslaunch.launch._unify_clear_params(
-                                                          config.clear_params):
+                    config.clear_params):
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py]: don't break line

File: rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py [hunk 12]
```diff
@@ -69,14 +69,14 @@ def load_parameters(config, caller_id):
...
                 if param_server.hasParam(caller_id, param)[2]:
                     param_server_multi.deleteParam(caller_id, param)
             r = param_server_multi()
             for code, msg, _ in r:
                 if code != 1:
                     raise RLException("Failed to clear parameter {}: ".format(
-                                                                         msg))
+                        msg))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py]: don't break line

File: rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py [hunk 13]
```diff
@@ -95,7 +95,7 @@ def load_parameters(config, caller_id):
             for code, msg, _ in r:
                 if code != 1:
                     raise RLException("Failed to set parameter: " +
-                                                "%s" % (msg))
+                                      "%s" % (msg))
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py]: don't break line

File: rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py [hunk 14]
```diff
@@ -149,7 +149,7 @@ def list_files(package, subdir, file_extension='.launch'):
 
         return [genmsg.resource_name(package, t)
                 for t in RqtRoscommUtil._list_types(
-                                                 path, file_extension)]
+            path, file_extension)]
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py]: don't break line

File: rqt_py_common/src/rqt_py_common/topic_completer.py [hunk 15]
```diff
@@ -57,7 +57,8 @@ def update_topics(self):
             topic_name = topic_path.strip('/')
             message_class = roslib.message.get_message_class(topic_type)
             if message_class is None:
-                qWarning('TopicCompleter.update_topics(): could not get message class for topic type "%s" on topic "%s"' % (topic_type, topic_path))
+                qWarning('TopicCompleter.update_topics(): could not get message class for topic type "%s" on topic "%s"' %
```
Comment (brawner) [inline rqt_py_common/src/rqt_py_common/topic_completer.py]: break into more lines

File: rqt_py_common/test/test_rqt_ros_graph.py [hunk 16]
```diff
@@ -63,7 +63,7 @@ def setUp(self):
         self._model.appendRow(node1)
 
         #node_list = [node1, node1_1, self._node1_1_1, node1_1_2, node1_2]
-        #self._model.appendRow(node_list)
+        # self._model.appendRow(node_list)
```
Comment (brawner) [inline rqt_py_common/test/test_rqt_ros_graph.py:67]: commented code

File: rqt_py_common/test/test_rqt_ros_graph.py [hunk 17]
```diff
@@ -78,9 +78,9 @@ def test_get_upper_grn(self):
 
     def test_get_lower_grn_dfs(self):
         self.assertEqual(len(RqtRosGraph.get_lower_grn_dfs(
-                                                  self._node1_1.index(),
-                                                  '')),
-                         self._len_lower_grn_node1_1)
+            self._node1_1.index(),
```
Comment (brawner) [inline rqt_py_common/test/test_rqt_ros_graph.py]: align with .get_lower_grn_dfs

File: rqt_gui/src/rqt_gui/ros_plugin_provider.py (hunk 1 above)
Comment (mlautman) [reply rqt_gui/src/rqt_gui/ros_plugin_provider.py:86]: I would prefer to keep changes limited to autopep8

File: rqt_py_common/src/rqt_py_common/rosaction.py (hunk 4 above)
Comment (mlautman) [reply rqt_py_common/src/rqt_py_common/rosaction.py:393]: Not actually required for flake8 or pep8

**Comment by mlautman** [comment]
@brawner, I fixed up the changes you requested and also changed the line length limit to 120 as per ROS style guild lines. If these changes look good I would really like to merge them and move on. We can always apply additional stylistic guidelines and changes later on

**Comment by brawner** [comment]
@mlautman Yah, let's move forward with porting code

**Review by dirk-thomas — APPROVED** [review]
LGTM

**Pull request merged**